content/api/rest-api.textile

```diff
@@ -626,7 +624,15 @@ bc[json]. {
   registrationToken: <string>
 }
 
-<!--[!!Ed. note: removed as part of EDU-1155. Return if required in the future.]For web:bc[json]. {  targetUrl: <string>  encryptionKey: <string>}-->
+For web:
+
+bc[json]. {
+  targetUrl: <string>,
+  encryptionKey: {
+    p256dh: <string>,
+    auth: <string>
```

Comment:
@owenpearson does this now need to include the public key?

Reply:
good point, will update